### ISSUE TITLE
Unable to clear color picker selection, or select transparent color

### ISSUE THREAD

**Issue by NiclasHorstad-acn** [magnolia-community/color-picker-field#3]
This might be more of a "help me out here" request rather than an actual issue...

In an _optional_ dialog color picker input, there is currently no way to either:

1. Clear the color selection (leave field with null value)
2. Select 'transparent' as the color of choice

Either of these two options would be appropriate, but after experimenting, I cannot find a way to achieve this.

Any help would be greatly appreciated.

**Comment by rgange** [comment]
Thanks for the feedback. Sorry for the slow response. I'll try to take a look at that soon.

**Comment by rgange** [comment]
Yes, you are right. There isn't a way to do such a thing. I will have to create a clear selection button or something similar.
